examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/PackageCardAdapter.kt

```diff
@@ -48,19 +79,57 @@ class PackageCardAdapter(
                 currentPackage.packageType.toString()
             }
 
-            binding.packageDetailsJsonObject.detail = ""
-                // TODOBC5 currentPackage.product.originalJson.toString(JSON_FORMATTER_INDENT_SPACES)
+            binding.packageDetailsJsonObject.detail = product.googleProduct?.productDetails?.toString() ?: "TODO Amazon"
+
+            bindPurchaseOptions(product)
 
             binding.root.setOnClickListener {
                 with(binding.packageDetailsContainer) {
                     visibility = if (visibility == View.GONE) View.VISIBLE else View.GONE
                 }
             }
         }
+
+        @SuppressLint("SetTextI18n")
+        private fun bindPurchaseOptions(product: StoreProduct) {
+            binding.packagePurchaseOptionGroup.removeAllViews()
+            val numberOfPurchaseOptions = product.purchaseOptions.size
+            product.purchaseOptions.forEach { purchaseOption ->
+                val radioButton = RadioButton(binding.root.context).apply {
```

Comment:
We could do this in a recycler view, but I believe we wouldn't have that many offers for the same plan? We can iterate on this and change it later if we want to make it more versatile.